### ISSUE TITLE
use_starter_package() fails badly

### ISSUE THREAD

**Issue by gadenbuie** [GerkeLab/grkmisc#3]
```
● Run `devtools::check()` to trigger spell check
✔ Adding 'magrittr' to Imports field in DESCRIPTION
✔ Writing 'R/utils-pipe.R'
● Run `devtools::document()`
Error in enc2utf8(path) : argument is not a character vector
```

**Comment by gadenbuie** [comment]
Possibly related to or caused by `https://github.com/r-lib/usethis/issues/571`

**Comment by gadenbuie** [comment]
Fixed in 4b4e4c5e8a65bc69a33a4c0fab751316549a43f5
